doc/riverctl.1.scd

```diff
@@ -196,6 +196,14 @@ that tag 1 through 9 are visible.
 *outer-padding* _pixels_
 	Set the padding around the edge of the screen to _pixels_.
 
+*unmap* [-release] _mode_ _modifiers_ _key_
```

Comment:
I have mixed feelings about `-release`.
It looks like an optional argument to `unmap` (well it kind of is) but if we had other arguments that do not specify the mapping to unmap further then it would look wired.
On the plus side the syntax is the same as with `map`.

What are your thoughts?

Reply:
I'm ok with how this is as it parallels `map` nicely. I also can't think of a reasonable alternative, do you have something else in mind?

Reply:
No I don't. lets keep it as it is!